packages/vite/src/node/server/index.ts

```diff
@@ -963,7 +945,7 @@ export async function _createServer(
     httpServer.listen = (async (port: number, ...args: any[]) => {
       try {
         // ensure ws server started
-        hot.listen()
```

Comment:
Let's have `Object.values(environments).forEach(e => e.hot.listen())` to have the same behaviour for now